audio/beets/Portfile

```diff
@@ -84,6 +84,7 @@ if {${name} eq ${subport} || "${name}-devel" eq ${subport}} {
 
     if {${name} eq ${subport}} {
         depends_lib-append \
+                    port:py${python.version}-gmusicapi \
```

Comment:
It looks like this is a duplicate, as you're also adding `py-gmusicapi` as a dependency on line 67.

Reply:
@mascguy indeed. Because I mean `depends_lib-delete`